api/v1alpha3/conversion_test.go

```diff
@@ -40,17 +41,18 @@ func TestFuzzyConversion(t *testing.T) {
 		Scheme:      scheme,
 		Hub:         &nextver.VSphereCluster{},
 		Spoke:       &VSphereCluster{},
-		FuzzerFuncs: []fuzzer.FuzzerFuncs{overrideDeprecatedFieldsFuncs},
+		FuzzerFuncs: []fuzzer.FuzzerFuncs{overrideVSphereClusterDeprecatedFieldsFuncs},
 	}))
 	t.Run("for VSphereMachine", utilconversion.FuzzTestFunc(utilconversion.FuzzTestFuncInput{
 		Scheme: scheme,
 		Hub:    &nextver.VSphereMachine{},
 		Spoke:  &VSphereMachine{},
 	}))
 	t.Run("for VSphereMachineTemplate", utilconversion.FuzzTestFunc(utilconversion.FuzzTestFuncInput{
-		Scheme: scheme,
-		Hub:    &nextver.VSphereMachineTemplate{},
-		Spoke:  &VSphereMachineTemplate{},
+		Scheme:      scheme,
+		Hub:         &nextver.VSphereMachineTemplate{},
+		Spoke:       &VSphereMachineTemplate{},
+		FuzzerFuncs: []fuzzer.FuzzerFuncs{CustomObjectMetaFuzzFunc},
```

Comment:
nit: Can we use the `CustomObjectMetaFuzzFunc` function from CAPI directly?

Reply:
it's not importable, I'll open an issue to move it to some util